source/sessions/tests/README.rst

```diff
@@ -37,10 +37,208 @@ Prose tests
 * ``client.startSession(snapshot = true, causalConsistency = true)``
 * Assert that an error was raised by driver
 
+2. Pool is LIFO
```

Comment:
Since the README already contained a numbered prose tests, the relocated tests start at 2.